src/core/Object3D.js

```diff
@@ -511,9 +511,7 @@ class Object3D extends EventDispatcher {
 
 		this.updateWorldMatrix( true, false );
 
-		const e = this.matrixWorld.elements;
-
-		return target.set( e[ 8 ], e[ 9 ], e[ 10 ] ).normalize();
+		return target.fromArray( this.matrixWorld.elements, 8 ).normalize();
```

Comment:
I'm not really a fan of these kind of refactoring since it the new code isn't really "better". Subjective code style changes can be the source of endless discussions. In general, it's best to avoid them in principle.

Reply:
alright